apps/meteor/client/components/SortList/ViewModeList.tsx

```diff
@@ -6,6 +6,7 @@ import ListItem from '../Sidebar/ListItem';
 
 const style = {
```

Comment:
I suggest removing this inline style and extending the component to receive these props, if the design team wants this flexibility on the title